src/test/java/net/pms/test/formats/FormatRecognitionTest.java

```diff
@@ -78,7 +85,7 @@ public void testRendererConfigurationBasics() {
     	// This test is only useful if the MediaInfo library is available
 		assumeTrue(mediaInfoParserIsValid);
 		RendererConfiguration conf = RendererConfiguration.getRendererConfigurationByName("Playstation 3");
-		assertNotNull("Renderer named \"Playstation 3\" found.", conf);
+		assertNotNull("Renderer named \"Playstation 3\" not found.", conf);
```

Comment:
@valib why did these need to change? I'm trying to review this, since the testing has gone well

Reply:
@SubJunk because this is the correct message when the `assertNotNull` returns error.